### PR TITLE
P0 buttons

### PR DESCRIPTION
**Pull request by stutisk**
https://user-images.githubusercontent.com/48529714/153244657-69ffbef8-514d-4036-954b-072d1ae90143.mp4

### CODE REVIEW

File: components/buttons/button.html [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    <title>Buttons</title>
+</head>
+
+<body>
+    <div class="container">
+        <nav class="nav-bar">
+            <div class="left-side">
+                <a href="../../index.html">GALA<span class="span">X!A</span></a>
+
+                <span class="bar"></span>
```
Comment (SaurabhChawda) [inline components/buttons/button.html]: blank Span ??

**Review by SaurabhChawda — APPROVED** [review]

File: components/buttons/button.html [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+                            <a href="">rating</a>
+                            <a href="">Toast</a>
+                            <a href="">simplified grid</a>
+                        </nav>
+
+                    </aside>
+                </div>
+            </div>
+            <!-- content -->
+            <div class="content">
```
Comment (adith-a-danthi) [inline components/buttons/button.html:74]: You can use `main` tag here

**Review by adith-a-danthi — APPROVED** [review]
LGTM! Try using semantic tags where possible.

**Comment by stutisk** [comment]
changes made ! thnku

**Pull request merged**